minifi-api/include/minifi-cpp/controllers/RecordSetReader.h

```diff
@@ -30,6 +30,12 @@ namespace org::apache::nifi::minifi::core {
 
 class RecordSetReader : public virtual controller::ControllerService {
  public:
+  static constexpr auto ControllerServiceApiDefinition = core::ControllerServiceApiDefinition{
```

Comment:
something like "ProvidesApi" would be a better name

come to think about it, I would also rename "ApiImplementations" to "ImplementsApis", but I feel less strongly about that

Reply:
👍 you wanna rename only the member or the struct aswell?
renamed the member 
https://github.com/apache/nifi-minifi-cpp/pull/2001/commits/b714eb903d50d1359832004904777dd6d296a912

Reply:
👍 good catch, I've added a test that would have failed https://github.com/apache/nifi-minifi-cpp/pull/2001/commits/ded65512ca5985a1d92ff26b4b995333f97a2550 and fixed it in https://github.com/apache/nifi-minifi-cpp/pull/2001/commits/2eacb4478f9306af31c0ce3a0a537384914d94ce